TripKit/Classes/helpers/TKAccessibilityHelper.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  TKAccessibilityHelper.swift
+//  TripKit-iOS
+//
+//  Created by Kuan Lun Huang on 22/2/18.
+//  Copyright © 2018 SkedGo. All rights reserved.
+//
+
+import Foundation
+import RxSwift
+
+@objc public class TKAccessibilityHelper: NSObject {
```

Comment:
Please provide some documentation for what this does and what’s the typical use case.

Reply:
Done